### ISSUE TITLE
Verify forward migration to IndexedDB works

### ISSUE THREAD

**Issue by 6a68** [mozilla-services/screenshots#4651]
(Related to #4534.) @rpl from the addons team [pointed out](https://bugzilla.mozilla.org/show_bug.cgi?id=1474696#c2) that the migration code isn't present in release (currently Firefox 61); the code landed in 62.

He suggested the following steps to verify that JSON-to-IndexedDB migration works as intended for Screenshots:

- create a new profile on release (Firefox 61)
- store a screenshot from the new profile while running release
- exit Firefox release
- open the same profile on nightly or beta (Firefox 63 or Firefox 62)
- switch the pref "ExtensionStorageIDB.enabled" to true
- exit Firefox
- restart firefox on the same profile (with Nightly or Beta, based on which version is being tested, possibly without reusing the same profile with both Nightly and Beta)
- open Screenshot again and verify that the "Screenshot Onboarding Tour" is not being shown again,
  then open "My shots" to verify that the previously stored shot is listed
- open the browser console and verify that the following log messages have been logged:
  ```
  Migrating storage.local data for ... (Screenshot debugName here)
  storage.local data successfully migrated to IDB Backend for ... (Screenshot debugName here)
  ```

**Comment by Softvision-CristinaBadescu** [comment]
I have tested this issue and it worked as expected: I was able to see the saved screenshot on Firefox Nightly 63.0a1 and the "Screenshots Onboarding Tour" was not displayed on the Nightly instance. 
I saw the log message only if I started the browser with browser console opened at the startup and only on Windows 10 x64. The message did not appear on Mac 10.13.5 and Arch Linux 4.12 x64.
